src/App.vue

```diff
@@ -7,7 +7,7 @@ provide("userService", new UserService());
 </script>
 
 <template>
-  <RouterLink to="/users">Users</RouterLink>
+<!--  <RouterLink to="/users">Users</RouterLink>-->
```

Comment:
remember to out-comment it when merging, to hide the navbar when signIn/signUp you can add v-if store.loggedUser and then it won't be visible if the user isn't logged in :)